c/extras/aggregator.cc

```diff
@@ -127,7 +126,7 @@ void Aggregator::aggregate_exemplars(DataTable* dt_exemplars,
   cols_counts[0] = Column::new_data_column(SType::INT32,
                                            static_cast<int64_t>(gb_members.ngroups()));
   cols_counts[1] = nullptr;
-  dt_counts = new DataTable(cols_counts);
+  dt_counts = new DataTable(cols_counts, {"counts"});
   auto d_counts = static_cast<int32_t*>(dt_counts->columns[0]->data_w());
   std::memset(d_counts, 0, static_cast<size_t>(gb_members.ngroups()) * sizeof(int32_t));
 
```

Comment:
Will probably need to change this part as already did some refactoring in the `aggregator-dai` branch. Thanks!

Reply:
The old `new DataTable(cols_counts);` will no longer compile -- this constructor became private.

Reply:
Sure, this is even better. Don't need to remember setting up the names. Will rebase my branch on top of `master` shortly.